python/ray/_private/ray_logging.py

```diff
@@ -186,11 +186,8 @@ def get_worker_log_file_name(worker_type, job_id=None):
     if job_id is None:
         job_id = os.environ.get("RAY_JOB_ID")
     if worker_type == "WORKER":
-        assert job_id is not None, (
-            "RAY_JOB_ID should be set as an env "
-            "variable within default_worker.py. If you see this error, "
-            "please report it to Ray's Github issue."
-        )
+        if job_id is None:
```

Comment:
How to handle duplicate PIDs?

Reply:
Not sure what `ray._private.worker.global_worker.worker_id` is -- is that a globally unique worker ID?

Reply:
If we have to debug an issue in the worker startup, can we easily map a worker with a job ID to the same worker without a job ID?

Reply:
yup `global_worker.worker_id` is unique.